src/resolveEnv.ts

```diff
@@ -10,19 +10,25 @@ import {
   string,
 } from "https://esm.sh/superstruct@0.15.4";
 
-const portFromString = defaulted(
-  coerce(
-    number(),
-    string(),
-    (value) => Number(value),
-  ),
-  8000,
-);
+function portFromString(defaultValue: number) {
+  return defaulted(
```

Comment:
I don't know if we need to validate the port range here?

Reply:
Yeah maybe overkill, leave it to the OS?